Issue with running slidev when `slides.md` is present!

I was exploring slidev and a leightwey to get it working is via docker if you do not want to have all the node setup. I followed this [guide](https://sli.dev/guide/install.html#install-on-docker) and run:

```sh
docker run --name slidev --rm -it \
    --user node \
    -v ${PWD}:/slidev \
    -p 3030:3030 \
    tangramor/slidev:latest
```
__Note__: The `slides.md` file does exist at ${PWD}.

When you run it, it throws this error: 

```sh
➜  presentation docker run --name slidev --rm -it \
    --user node \
    -v ${PWD}:/slidev \
    -p 3030:3030 \
    tangramor/slidev:latest

npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /slidev/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/slidev/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2023-11-16T21_29_14_736Z-debug-0.log
Start slidev...
npm notice 
npm notice New major version of npm available! 9.5.1 -> 10.2.4
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.2.4
npm notice Run npm install -g npm@10.2.4 to update!
npm notice 
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/slidev - Not found
npm ERR! 404 
npm ERR! 404  'slidev@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2023-11-16T21_29_15_436Z-debug-0.log
```

I didn't go into too many details about why this is coming, but looked into entrypoint and found out that the issue is mainly from this: https://github.com/tangramor/slidev_docker/blob/9177284115a4a943f1c5f73b6266f206f106333b/entrypoint.sh#L9

I did fix it and was able to run it: 
```sh
➜  presentation docker run --name slidev --rm -it \
    --user node \
    -v ${PWD}:/slidev \
    -p 3030:3030 \
    slidev

up to date in 3m
npm notice 
npm notice New major version of npm available! 9.8.1 -> 10.2.4
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.2.4
npm notice Run npm install -g npm@10.2.4 to update!
npm notice 
Start slidev...


  ●■▲
  Slidev  v0.44.0 

  theme       @slidev/theme-default
  css engine  unocss
  entry       /slidev/slides.md

  public slide show   > http://localhost:3030/
  presenter mode      > http://localhost:3030/presenter/
  remote control      > http://172.17.0.2:3030/entry/

  shortcuts           > restart | open | edit | quit | qrcode
```

I will push the fix via PR.

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -6,7 +6,7 @@ if [ -f /slidev/slides.md ]; then
     if [ -d /slidev/node_modules ]; then
         npm update
     else
-        npm install
+        npm install @slidev/cli @slidev/theme-default @slidev/theme-seriph
     fi
     echo "Start slidev..."
     npx slidev --remote

